src/cellfinder_core/__init__.py

```diff
@@ -1,3 +1,7 @@
 __version__ = "0.3.0"
 __author__ = "Adam Tyson, Christian Niedworok, Charly Rousseau"
 __license__ = "BSD-3-Clause"
+
+import logging
```

Comment:
I don't know what the "standard" way of doing it in `__init__.py`. Should the import be here or at the top?

Reply:
🤷 I'm not sure but isort isn't complaining, so I guess this is fine?